From f1a7cfcb1d487c19c27287fe5136c46f3b7de1ab Mon Sep 17 00:00:00 2001
From: Matti Niemenmaa <matti.niemenmaa+git@iki.fi>
Date: Mon, 10 Sep 2018 15:21:31 +0300
Subject: [PATCH 386/686] fallocate: add missing semicolon

This broke compilation when HAVE_POSIX_FALLOCATE was undefined. The typo
dates to the original posix_fallocate support added in commit
833f9a7aae713278eec5f85266597482f18c7370.

Signed-off-by: Matti Niemenmaa <matti.niemenmaa+git@iki.fi>
---
 sys-utils/fallocate.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sys-utils/fallocate.c b/sys-utils/fallocate.c
index 48a1b27..ba3867c 100644
--- a/sys-utils/fallocate.c
+++ b/sys-utils/fallocate.c
@@ -352,7 +352,7 @@ int main(int argc, char **argv)
 			posix = 1;
 			break;
 #else
-			errx(EXIT_FAILURE, _("posix_fallocate support is not compiled"))
+			errx(EXIT_FAILURE, _("posix_fallocate support is not compiled"));
 #endif
 		case 'v':
 			verbose++;
-- 
1.8.3.1

